app/waf/src/models/declaration.model.ts

```diff
@@ -29,3 +29,19 @@ export class Declaration extends CommonEntity {
     super(data);
   }
 }
+
+export class ASGDeployRequest extends Entity {
```

Comment:
Does it need to inherits from Entity?

Reply:
It's used to generate the openapi schema, during which @property can decorate only Entity's properties. 

It will report error like this if no `extends Entity`: 
```src/controllers/declaration.controller.ts:359:9 - error TS2345: Argument of type 'typeof ASGDeployRequest' is not assignable to parameter of type 'typeof Entity'.
  Type 'typeof ASGDeployRequest' is missing the following properties from type 'typeof Entity': getIdOf, buildWhereForId, modelName, definition```

Reply:
Is it an action request body which includes ADC id to specify target device? If true, I got its meaning. Thanks!